src/bootstrap/src/core/download.rs

```diff
@@ -852,7 +852,8 @@ download-rustc = false
             t!(fs::create_dir_all(&gcc_cache));
         }
         let base = &self.stage0_metadata.config.artifacts_server;
-        let filename = format!("gcc-nightly-{}.tar.xz", self.build.triple);
+        let version = self.artifact_version_part(gcc_sha);
+        let filename = format!("gcc-{version}-{}.tar.xz", self.build.triple);
```

Comment:
Will this work fine on stable channels?

https://github.com/rust-lang/rust/blob/b10555674f355aca2bfe974e50a0b9ab48eb1d87/src/bootstrap/src/core/config/config.rs#L2704-L2709

Reply:
We use the exact same code for downloading LLVM and CI rustc, and the GCC dist component should produce the dist artifacts on all channels, so I think it should. But as usually, it's hard to be sure until we try.. :laughing: